### ISSUE TITLE
Difference between tap number and juts moving cursor is not accurate

### ISSUE THREAD

**Issue by encrustace** [iamkroot/asus-numpad#14]
When trying to move the cursor it types the number while Numpad is enabled

**Comment by iamkroot** [comment]
Yep, this is a known limitation. Sadly, it is going to be _extremely_ difficult to fix this. I have been trying to tackle some of it at https://github.com/iamkroot/asus-numpad/tree/smart-grab but it is still pretty finicky. Would appreciate your feedback on it. To install, you'll need to follow the instructions under "Manual install" heading in README.

**Comment by encrustace** [comment]
I will try it today and will update the result here

**Comment by encrustace** [comment]
It is better than before, if I try to move the cursor on the calculator opened out of 10 times it clicks the number 4 times, if the move is super quick then it works smoothly else it clicks the number.

**Comment by encrustace** [comment]
I have implemented some changes, now touch and cursor move is very accurate
